### DISCUSSION TITLE
Can we detect non moving objects?

### DISCUSSION THREAD

**Discussion by R5fan** [blakeblackshear/frigate#1817] [Ideas]
In home assistance we get a sensor.camera.object entity that counts the number of those objects for that camera or region, but it resets to zero almost instantly when no more movement is detected, even if the object is still there.

I understand frigate only runs inference on motion , which makes complete sense, but I would occassionally like to check a camera or region for the number of objects that are there now, even if nothing is moving.   Say number of cars on my driveway or people or (sleeping) dogs in my garden. I can try to keep track of that by relying on motion and assuming objects move in and out of zones, but this breaks quickly as not every motion is always perfectly detected, and that dog that entered the garden can leave it as a cat so to speak. or the view may be blocked by something. Also when i restart my server, I lose count. 

An API call to periodically run inference on a camera or region even with no movement, and getting a "static" object count would be super handy!

**Comment by blakeblackshear** [comment]
Duplicate https://github.com/blakeblackshear/frigate/issues/841
